spec/factories/tutor.rb

```diff
@@ -20,6 +20,8 @@
     race { ApplicationHelper.race.sample[0] }
     smartt_id { Faker::Base.numerify('####-######') }
     state { ApplicationHelper.us_states.sample[0] }
+    training_date { Faker::Date.between(1.year.ago, 1.day.ago) }
```

Comment:
It doesn't really matter. There's no validation around the date. But we can change it if you like.